FS19/ML/Zusammenfassung_ML.tex

```diff
@@ -2508,13 +2508,14 @@ \subsection{Base Transformation on Data Matrices}
 \subsection{Eigenvectors and Eigenvalues}
 
 \begin{description}
-    \item[Eigenvector: ] A vector that does only stretch or rotate (but noth both) a matrix $A$ when multiplied with the matrix
-    \item[Eigenvalue: ] A factor of the eigenvector
+    \item[Eigenvector: ] A vector that is a only a multiple of itself (i.e. stretches) when multiplied with the matrix $A$, no rotation!
```

Comment:
Lose the 'no rotation' - that was purely for your information

Reply:
I think it's useful to emphasise that it does purely stretch the vector